### DISCUSSION TITLE
Is a new release planned for sometime soon?

### DISCUSSION THREAD

**Discussion by mitchjust-finocomp** [testcontainers/testcontainers-java#4548] [Q&A] [answered]
I'm hoping to get the fix merged in https://github.com/testcontainers/testcontainers-java/pull/4388, but no release has been done since. Any idea when to expect one? Thanks!

**Comment by kiview** [comment, marked as answer]
You can expect a release very soon, definitely within this month.

**Reply by kiview** [reply]
1.16.1 has been released yesterday 🙂 
https://github.com/testcontainers/testcontainers-java/releases/tag/1.16.1

**Reply by mitchjust-finocomp** [reply]
Thanks @kiview!
